HTShowcase/HTShowcase/ViewControllers/Views/ShowcaseSquaredCollectionViewCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  ShowcaseSquaredCollectionViewCell.swift
+//  HTShowcase
+//
+//  Created by Gonzalo Barrios on 5/23/18.
+//  Copyright © 2018 Gonzalo Barrios. All rights reserved.
+//
+
+import UIKit
+
+class ShowcaseSquaredCollectionViewCell: UICollectionViewCell {
+    
+    static let cellReuseIdentifier = "ShowcaseSquaredCollectionViewCellIdentifier"
+    static let cellNibName = "ShowcaseSquaredCollectionViewCell"
+    
+    @IBOutlet weak var containerView: UIView!
+    @IBOutlet weak var appImageView: UIImageView!
+    @IBOutlet weak var appNameLabel: UILabel!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        backgroundColor = .clear
+    }
+    
+    func setup(withApp app: App) {
+        self.appImageView.setImage(withStringURL: app.imageURL)
+        self.appNameLabel.text = app.name
+        self.appImageView.layer.masksToBounds = true
+        
+        containerView.layer.cornerRadius = 7
+        let shadowPath2 = UIBezierPath(rect: self.bounds)
```

Comment:
Smae bout shadowPath2, and also, is it not possible to create a class for the cell that does this? Since it is used in 3 different cells it might be good to do it